CONTRIBUTING.md

```diff
@@ -47,6 +47,30 @@ paper and understand how it fits into the TensorFlow community. This
 process can take longer than typical commit reviews so please bare with
 us**
 
+## Colab Support
+
+You can add examples using both Jupyter Notebook and Colab Notebooks.
+
+Want to add more examples using COLAB?
+
+Follow this:
+
+1) Open [Colab](https://colab.research.google.com/)
+2) Add a new example in the [addons/examples](https://github.com/tensorflow/addons/tree/master/examples) directory.
+Template to write a example can be found [here](https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb).
+
+3) Open File->Save a copy in Github.
+4) Authorize your Colab with Github.
+5) Save it in the desired branch.
```

Comment:
Close! I think the following steps are needed:

1) Create a branch on your fork

2) Goto colab.google.com and start a new notebook using this template:
https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb

3) Remove the "View on TensorFlow.org Link"

4) Edit the "View source on Github" and "Run in Google Colab" links so that they point to where the link will reside after it's merged into master (e.g. "https://colab.research.google.com/github/tensorflow/addons/blob/master/examples/layers_weightnormalization.ipynb")

5) Install TF2 and tensorflow-addons
!pip install tensorflow==2.0.0.a0
!pip install tensorflow-addons

6) Follow the guidelines of the template

7) Save a copy in github on your branch